programming/sr/vision/markers.md

```diff
@@ -18,9 +18,9 @@ Let's break that down:
 
 Details of the types and size of markers used in the SR2019 game can be found in the [rules](/docs/rules).
 
-You can download all of the markers as a single [ZIP file](/docs/resources/2019/sr-dev-markers-sr2019.zip)
-or individually from the [git repo](https://github.com/srobo/game-markers/tree/master/SR2019/dev).
-The *Arena* markers, due to their size, need to be printed on A3 paper.
+You can download all of the markers as a single [ZIP file](/docs/resources/2020/sr-dev-markers-sr2020.zip)
```

Comment:
I notice this points to a ZIP which doesn't exist yet?

Reply:
Good timing, I just remembered this and came here to add it to this PR!

Reply:
652115f